examples/custom_window_frame/src/main.rs

```diff
@@ -104,6 +98,38 @@ fn custom_window_frame(
                 frame.drag_window();
             }
 
+            // Add the close button:
+            let close_response = ui.put(
+                Rect::from_min_size(rect.left_top(), Vec2::splat(height)),
+                Button::new(RichText::new("❌").size(height - 4.0)).frame(false),
+            );
+            if close_response.clicked() {
+                frame.close();
+            }
+
+            let minimized_response = ui.put(
+                Rect::from_min_size(
+                    rect.left_top() + vec2((height - 4.0) * 1.0, 0.0),
+                    Vec2::splat(height),
+                ),
+                Button::new(RichText::new("--").size(height - 4.0)).frame(false),
```

Comment:
The default egui fonts have icons for minimize and maximize:

```suggestion
                Button::new(RichText::new("🗕").size(height - 4.0)).frame(false),
```